libs/application/ui-components/src/components/ExportAsCSV/ExportAsCSV.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Box, Button } from '@island.is/island-ui/core'
+import { CSVLink } from 'react-csv'
+
+interface ExportCSVProps {
+  data: string | object[]
+  filename: string
+  title: string
+}
+
+export const ExportAsCSV = ({
+  data,
+  filename,
+  title = 'Sækja niðurstöður',
+}: ExportCSVProps) => {
+  return (
+    <Box display="flex">
+      <CSVLink
+        data={data && data.length > 0 ? data : 'Engin gögn fundust'}
```

Comment:
Would be nice to not have hardcoded strings here and pass them as props so we can use formatMessage in parent component

Reply:
Yes, moved to props